wet-boew/ajax-fetch/ajax-fetch-en.html

```diff
@@ -0,0 +1,32 @@
+---
+{
+	"title": "ajax-fetch",
+	"language": "en",
+	"description": "Handles Ajax requests and triggers events for success or failure.",
+	"altLangPage": "ajax-fetch-fr.html",
+	"dateModified": "2024-08-01"
+}
+---
+<ul>
+	<li><a class="btn btn-primary" href="https://github.com/wet-boew/GCWeb/issues/new?title=Ajax%20Fetch">Questions or comments?</a></li>
+</ul>
```

Comment:
I don't believe this is necessary as they are not actual plugins. Do this for all your examples.
```suggestion
```